spec/controllers/facility_statements_controller_spec.rb

```diff
@@ -30,10 +30,14 @@ def create_order_details
 
     before(:each) do
       @authable = FactoryGirl.create(:facility)
+      @other_facility = FactoryGirl.create(:facility)
       @user = FactoryGirl.create(:user)
+      @other_user = FactoryGirl.create(:user)
       UserRole.grant(@user, UserRole::ADMINISTRATOR)
       @account = FactoryGirl.create(@account_sym, account_users_attributes: account_users_attributes_hash(user: @user), facility_id: @authable.id)
+      @other_account = FactoryGirl.create(@account_sym, account_users_attributes: account_users_attributes_hash(user: @other_user), facility_id: @other_facility.id)
```

Comment:
Do we need these additional objects in all of the tests or can we make them more local to the tests that specifically need them?